css/index.css

```diff
@@ -101,21 +147,71 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+  .box {
     width: 12.5%;
     height: 100px;
     background: black;
-    margin: 20px 2.5%;
     color: white;
+    margin: 20px 2.5%;
+   font-size:1.6rem;
     display: flex;
     align-items: center;
     justify-content: center;
+}
+ #box1 {
+color: white;
+background:black;
 }
 
+ #box2 {
+
+color:white;
+background:forestgreen;
+}
+#box3 {
+color: white;  
+background: black;
+}
+ #box4 {
+ color: white;     
+background:darkorchid;
+}
+#box5 {
+color: white;   
+background: black;
+}
+#box6 {
+  color: white;   
+background:hotpink;
+}
+#box7 {
+  color: white;   
+background: black;
+}
+#box8 {
+  color: white;   
+background:indigo;
+}
+#box9 {
+   color: white;  
+background: black;
+}
+#box10{
+  color: white;   
+background:lawngreen;
```

Comment:
Since the `color` attribute on all of these are the same, thus we could add this to the `.box` class.